### PR TITLE
[Storage] Validate errors on create bucket modal test

### LINKED ISSUE #2221: [Storage] Add validate errors when trying to create a bucket test
As described add coverage when creating a bucket:

- User cannot create a bucket with no name
- User cannot create a bucket with disallowed characters (eg space only)

### PR DESCRIPTION
**Pull request by juans-chainsafe**
closes #2221

### CODE REVIEW

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 1]
```diff
@@ -21,16 +21,32 @@ describe("Bucket management", () => {
       createBucketModal.cancelButton().click()
       createBucketModal.body().should("not.exist")
 
-      // create a bucket and see it in the bucket table
+      // go to create bucket modal
       bucketsPage.createBucketButton().click()
       createBucketModal.body().should("be.visible")
+      // ensure can't create an empty bucket
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts:28]: Minor thing but I find that leaving a whitespace before each comment / individual check in the flow of the test helps readability, it's not done here but is elsewhere in the test.

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 2]
```diff
@@ -21,16 +21,32 @@ describe("Bucket management", () => {
...
+      createBucketModal.submitButton().click()
+      createBucketModal.bucketNameInput().should("have.class", "error")
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts:30]: Interesting, I've never thought about using the class name before like this. I would usually avoid doing that (as I'd be concerned about it being too generic and potentially picking up a false match from elsewhere on the page)...but you've scoped it to within the bucketNameInput element so I guess that's not a valid concern, nice! 🤩

**Review by Tbaut — APPROVED** [review]
:+1:

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts (hunk 1 above)
Comment (juans-chainsafe) [reply packages/storage-ui/cypress/tests/bucket-management.cy.ts:28]: yep! it helps, done!

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts (hunk 2 above)
Comment (juans-chainsafe) [reply packages/storage-ui/cypress/tests/bucket-management.cy.ts:30]: yes! the other way around is checking that the modal is still present and validating the text... but we don't want to add validations that have hardcoded text, so I ended up doing that inside the input, so we shouldn't have any problem. Also, I added a validation when the input is correct that the class 'error' should not be present

**Review by asnaith — APPROVED** [review]
Looks good! Thanks for this.

**Pull request merged**